use-case-zeit-now-ci-cd.md

```diff
@@ -0,0 +1,246 @@
+            - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+            - npm install
+
+            # Store the latest version of node modules in cache to reuse in
+            # further blocks:
+            - cache store client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json) node_modules
+
+  - name: Run tests
+    task:
+      jobs:
+        - name: npm test
+          commands:
+            - checkout
+            - nvm use
+            - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum package-lock.json),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+            - npm test
+
+promotions:
+  # Deployment to staging can be triggered manually:
+  - name: Deploy to staging
+    pipeline_file: deploy-staging.yml
+
+  # Automatically deploy to production on successful builds on master branch:
+  - name: Deploy to production
+    pipeline_file: deploy-production.yml
+    auto_promote_on:
+      - result: passed
+        branch:
+          - master
+```
+
+Just 2 blocks make the CI pipeline:
+
+![CI Pipeline](https://github.com/semaphoreci-demos/semaphore-demo-zeit-now/raw/master/images/semaphore-zeit-now-ci.png)
+
+-   npm install and cache:
+    -   Downloads and installs the Node.js packages.
+    -   Builds the app and saves it to the cache.
+-   npm test:
+    -   Runs unit and coverage tests.
+
+Two
+[promotions](https://docs.semaphoreci.com/article/50-pipeline-yaml#promotions)
+branch out of the CI pipeline:
+
+-   Deploy to production: automatically started once all tests are green
+    for the master branch.
+-   Deploy to staging: manually started from Semaphore's dashboard.
+
+Continuous deployment pipeline
+------------------------------
+
+``` yaml
+# .semaphore/deploy-production.yml
+
+version: v1.0
+name: Deploy to production
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Deploy to production
+    task:
+      secrets:
+        - name: now
+      jobs:
+      - name: Deploy to Zeit Now
+        commands:
+          - checkout
+          - nvm use
+          - npm install now -g
+          - now --token $ZEIT_TOKEN --local-config production.json
+```
+
+The deployment pipeline consists of one block:
+
+![CD Pipeline](https://github.com/semaphoreci-demos/semaphore-demo-zeit-now/raw/master/images/semaphore-zeit-now-cd-production.png)
+
+### Node.js
+
+Both
+[nvm](https://docs.semaphoreci.com/article/32-ubuntu-1804-image#javascript-via-node-js)
+and [npm](https://www.npmjs.com) are pre-installed on Semaphore's Ubuntu
+image. We can use the them to switch Node.js versions and install
+packages.
+
+### Deployment
+
+[Zeit Now](https://zeit.co/) is a cloud service for serverless websites
+and web applications. Deployment is done with their [command
```

Comment:
Let's refrain from saying "our" or "their" anywhere in our content. Simply say "Deployment is performed with the Now CLI ..."